.release-please-manifest.json

```diff
@@ -1,21 +1,21 @@
 {
-  "apps/cowswap-frontend": "1.64.3",
+  "apps/cowswap-frontend": "1.65.0",
   "apps/explorer": "2.29.2",
-  "libs/permit-utils": "0.1.2",
-  "libs/widget-lib": "0.7.1",
-  "libs/widget-react": "0.5.0",
-  "apps/widget-configurator": "1.0.2",
-  "libs/analytics": "1.0.0",
+  "libs/permit-utils": "0.2.0",
+  "libs/widget-lib": "0.8.0",
+  "libs/widget-react": "0.6.0",
```

Comment:
I'm already in [0.7.0-RC.0](https://www.npmjs.com/package/@cowprotocol/widget-react/v/0.7.0-RC.0)

https://www.npmjs.com/package/@cowprotocol/widget-react?activeTab=versions

Reply:
How was 0.7.0 RC released?

From the changes 
![image](https://github.com/cowprotocol/cowswap/assets/43217/700e689a-d219-4173-b7cb-ab54e4762ee1)

Looks like the previous version release-please knows about was 0.5.0, so 0.6.0 is the natural next one.

You can still edit this manually.

But then again, where is 0.6.0 if you have 0.7.0-rc?

Reply:
I released from a branch using https://github.com/cowprotocol/cowswap/actions/workflows/npm.yml